test/test_apidb_backend_changeset_uploads.cpp

```diff
@@ -161,9 +162,11 @@ TEST_CASE_METHOD( DatabaseTestsFixture, "test_single_nodes", "[changeset][upload
 
   SECTION("Create new node")
   {
+    test_request req{};
```

Comment:
I believe we could avoid the repeated  `test_request req{};  RequestContext ctx{req};` declarations here by moving these two lines to line 142 (right after TEST_CASE_METHOD.

During test execution, these "top level" declarations would be automatically re-executed for each SECTION, so that each SECTION would use a fresh ctx object.

In cases where sharing does not make sense (e.g. ctx2), I would leave them where they are today.

Reply:
Ah, yes you're right. Forgot you can do that with catch2.